language/sample-apps/chat-flask-cloudrun/README.md

```diff
@@ -6,6 +6,9 @@ This application demonstrates a Cloud Run application that has a simple Form-bas
 
 ## Build and Deploy the application to Cloud Run
 
+> NOTE: **Before you move forward, ensure that you have followed the instructions in [SETUP.md](../SETUP.md).**
+Additionally, ensure that you have cloned this repository and are currently in the ```chat-flask-cloudrun``` folder for the rest of the commands.
```

Comment:
In terms of phrasing -- could you clarify exactly what you mean by "currently in the ____ folder" so users know what to do to make sure they are currently in the folder? For example, "currently in the ____ folder and it is your active working directory for the rest of the commands" or something like that?

Reply:
Thanks. I have rephrased it now across all the READMEs in this PR.